ffi/rust/firedancer-sys/src/ballet/mod.rs

```diff
@@ -4,10 +4,12 @@ mod sha256;
 mod sha512;
 mod shred;
 mod txn;
+mod bmtree;
 
 pub use ed25519::*;
 pub use sbpf::*;
 pub use sha256::*;
 pub use sha512::*;
 pub use shred::*;
 pub use txn::*;
+pub use bmtree::*;
```

Comment:
Can you also add a newline at the end here :)

Reply:
> Can you also add a newline at the end here :)

yep done!